extensions/2.0/Vendor/EXT_lights_environment/README.md

```diff
@@ -0,0 +1,128 @@
+# EXT_lights_environment
+
+## Contributors
+
+* `TODO: Add contributors`
+
+## Status
+
+Draft
+
+## Dependencies
+
+Written against the glTF 2.0 spec.
+
+## Overview
+
+This extension provides the ability to define an environment light (also known as [sky light](https://docs.unrealengine.com/en-US/Engine/Rendering/LightingAndShadows/LightTypes/SkyLight/index.html)) in a glTF scene. In general, this environment is used for image based lighting in rasterizers and pathtracers.  
+
+To environment light is defined by a panorama image (also know as [equirectangular image](https://docs.blender.org/manual/en/latest/render/lights/world.html)). For encoding the panorama image and for supporting [High-dynamic-range rendering](https://en.wikipedia.org/wiki/High-dynamic-range_rendering), the widely used [RGBE image format](https://en.wikipedia.org/wiki/RGBE_image_format) is used as a `.hdr` file:  
+The panorama image and `.hdr` file format is supported in paint tools like Gimp or Photoshop, 3D modelling tools like Blender or 3ds Max, Game Engines like Unity or Unreal Engine and Web Engines like Babylon.js or three.js.  
```

Comment:
I think I would consider panorama / equirectangular IBL easy to create, but inefficient to load, which is the opposite of what we usually aim for with glTF. In Babylon.js and three.js the IBL would have to be converted to a cubemap at load time.

Other than wide support in paint tools, are there other reasons to prefer equirectangular? All of these engines should support cubemaps, too.

Reply:
We do need a) HDR and b) a common supported format. In general, KTX2 would be a perfect fit, but unfortunatelly it does not support HDR compression at this point of time.
I disagree, that .hdr is inefficient to load. It is easy to load (some lines of code and straight forward), nicely encoded and one can choose the final precission and resolution easily (e.g. half float vs. float) for the final cube map. With this preprocessing argument, we could not allow JPEG and/or Basisu compression, as there is also quite something happening on the CPU before the final image is uploaded to the GPU. It is in most cases just hidden in some library.
It is not just about paint tools. Any DCC (2D or 3D) tool can handle .hdr panorama files for IBL. Is there any out there, which does not support .hdr?
I understand your point of view, but some groups want to have this extension and/or feature set. That's the reason for EXT_ and we should not bring all on the same page, as it will not become a KHR_ extension.

Reply:
I'm not critiquing the choice of .hdr here, but the choice of panorama instead of cubemap. Couldn't you use 6 `.hdr` images to form a cubemap, or is that just unusual?

Reply:
That is unusual. E.g. in Blender you just define the panorma .hdr file:
![image](https://user-images.githubusercontent.com/27856744/90175156-49767b80-dda7-11ea-859c-3c09790e37d2.png)
So, for Blender, an easy import and export would be possible.

Reply:
> I think I would consider panorama / equirectangular IBL easy to create, but inefficient to load, which is the opposite of what we usually aim for with glTF. 

 I agree with @donmccurdy 
If we choose a cubemap format we will reduce the load time process - I think this is something we should aim for!